src/coreclr/build-runtime.sh

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env bash
```

Comment:
Is this script meant for direct developer usage as say `build.cmd` formerly was?

Reply:
Yes. For people who want to avoid using MSBuild as much as possible or for people who are building on platforms where we don't have a working MSBuild, they can call this script directly. It should have the same user experience as the old build.cmd/sh scripts except it only builds the CoreCLR runtime and cross components.